tests/scripts/all.sh

```diff
@@ -467,15 +467,15 @@ tests/scripts/doxygen.sh
 # 1.1 Basic tests: default config
 #################################
 
-msg "build: cmake, gcc, ASan" # ~ 1 min 50s
+msg "build: cmake, gcc, ASan" # ~ 1 min
```

Comment:
Where do these numbers come from? How are they measured? Is their benefit worth the maintenance effort?

Reply:
Those numbers come from the output of a successful run (`msg` prints current time) plus some math made by hand. (In theory, they're supposed to be added when adding a new test, but in practice that doesn't quite happen.)

Regarding benefit vs effort, now that you're asking, probably not. In the old times, `all.sh` used to be ordered by decreasing "perceived likelihood of finding bugs I missed during development" / "running time", but that stopped making sense a while ago. With the idea that we're probably going to rationalize our test campaigns soon, I thought it made sense to update them one last time in case the timings might guide use in that work, but in the long run I don't think those numbers are worth the effort (but if we think they are, we should make their computation more automated).